data.sql

```diff
@@ -1,5 +1,4 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals(id, name, date_of_birth, escape_attempts, neutered,weight_kg) values(1, 'Agumon', '2020-03-02', true, 0, 10.23);
```

Comment:
- [ ] Kindly note that you need to follow the order you declared to insert your data into the database when adding values. in your syntax you exchanged the `neutered` and the `escape_attempts` that why is throwing the error above :point_up: . Kindly fix it.

Reply:
Thank you @andersonlebon for suggesting these changes. Now, it's solved.